### PR TITLE
Add EL9 to Candlepin nightly

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: theforeman.org/pipelines/vars/candlepin/nightly.groovy [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 def packaging_branch = 'rpm/develop'
+def candlepin_distros = [
+    'el8',
+    'el9'
+]
 def pipelines = [
     'candlepin': [
-        'centos8-stream'
+        'centos8-stream',
+        'centos9-stream'
```
Comment (evgeni) [inline theforeman.org/pipelines/vars/candlepin/nightly.groovy:10]: This is currently unused (as we do not yet have CP tests), but would start failing once we do, or does this PR mean we also have el9 packages now, not only repos?

**Review by evgeni — APPROVED** [review]

File: theforeman.org/pipelines/vars/candlepin/nightly.groovy (hunk 1 above)
Comment (evgeni) [reply theforeman.org/pipelines/vars/candlepin/nightly.groovy:10]: https://copr.fedorainfracloud.org/coprs/g/theforeman/candlepin-nightly-staging/build/6888982/ seems like there are EL9 builds 🤘

Comment (ehelms) [reply theforeman.org/pipelines/vars/candlepin/nightly.groovy:10]: Means both -- I tested this before hand with nightly https://yum.theforeman.org/candlepin/nightly/el9/x86_64/

Comment (ehelms) [reply theforeman.org/pipelines/vars/candlepin/nightly.groovy:10]: And working puppet module tests -- https://github.com/theforeman/puppet-candlepin/pull/251

**Pull request merged**